tools/integrations/manage.sh

```diff
@@ -117,6 +117,15 @@ cmd_publish() {
     docker push "$versioned_image"
     docker push "$latest_image"
   fi
+  
+  # Checking if the image was successfully registered on DockerHub
+  # see the description of this PR to understand why this is needed https://github.com/airbytehq/airbyte/pull/11654/
+  sleep 5
+  TAG_URL="https://hub.docker.com/v2/repositories/${image_name}/tags/${image_version}"
+  DOCKERHUB_RESPONSE_CODE=$(curl --silent --output /dev/null --write-out "%{http_code}" ${TAG_URL})
```

Comment:
I wonder if this step would benefit from some retry?  does dockerhub fail in a way that a retry shortly after would be useful?  the man page for curl has a fairly deep set of retry options if that is interesting to you

cool use of pulling the HTTP code here ☝️